src/test/java/uk/ac/ebi/atlas/experimentpage/markergenes/MarkerGeneTest.java

```diff
@@ -0,0 +1,82 @@
+package uk.ac.ebi.atlas.experimentpage.markergenes;
+
+import com.google.common.collect.ImmutableSet;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+import uk.ac.ebi.atlas.controllers.BioentityNotFoundException;
+import uk.ac.ebi.atlas.search.CellTypeSearchDao;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
```

Comment:
Did you try running the tests without `LENIENT`?

Reply:
Removed and tested.

Reply:
Got some errors. Fixed them.